WORKSPACE

```diff
@@ -79,6 +79,60 @@ yarn_install(
     yarn_lock = "//:yarn.lock",
 )
 
+# Required for buildifier (`//scripts:buildifier`)
```

Comment:
Using https://github.com/keith/buildifier-prebuilt would remove the need for the additional go dependencies.

Reply:
It's pretty easy to avoid needing to build from source. I may just do that.

Reply:
Done!